apps/cms/package.json

```diff
@@ -37,9 +38,13 @@
     "@tietokilta/ui": "workspace:*",
     "@types/express": "^4.17.21",
     "@types/lodash": "^4.17.0",
+    "@types/papaparse": "^5.3.14",
     "@types/passport-oauth2": "^1.4.15",
+    "@types/react": "^18.2.65",
     "copyfiles": "^2.4.1",
+    "csvtojson": "^2.0.10",
```

Comment:
This dependency doesn't seem to be used anymore? If it's not used it should probably be removed